.travis.yml

```diff
@@ -10,6 +10,8 @@ install:
   - go get -u golang.org/x/lint/golint
 
 before_script:
+  - sudo mkdir -p /ipfs/keystore
+  - sudo chmod -R 777 /ipfs
```

Comment:
is this path fixed? can this be configured via a configuration option so that a non-root path can be used?

Reply:
Yup, it's hard-coded within the temporal code-base.

Reply:
actually it looks like it can be configured already: https://sourcegraph.com/github.com/RTradeLtd/Temporal@dc4e58eb9050ff2897a13493a2ec0c198a599a99/-/blob/rtfs/keystore.go#L25

actually, based on the [references](https://sourcegraph.com/github.com/RTradeLtd/Temporal@dc4e58eb9050ff2897a13493a2ec0c198a599a99/-/blob/rtfs/keystore.go#L25:6&tab=references), it looks like no tests are actually using the default path?

Reply:
Line 130 in `api.go` initializes the keystore manager with its IPFS connection, which uses a default path of `/ipfs/keystore` in `rtfs/utils.go`.

Reply:
We should make that part of API configuration then 👍